blog/electron-31-0.md

```diff
@@ -0,0 +1,88 @@
+
+## Notable Changes
+
+### Highlights
+
+- Extended `WebContentsView` to accept pre-existing `webContents` object. [#42319](https://github.com/electron/electron/pull/42319)
+- Added support for `NODE_EXTRA_CA_CERTS`. [#41689](https://github.com/electron/electron/pull/41689)
+- Updated window.flashFrame(bool) to flash continuously on macOS. [#41391](https://github.com/electron/electron/pull/41391)
+- Removed `WebSQL` support [#41868](https://github.com/electron/electron/pull/41868)
+- `nativeImage.toDataURL` will preserve PNG colorspace [#41610](https://github.com/electron/electron/pull/41610)
+- Extended `webContents.setWindowOpenHandler` to support manual creation of BrowserWindow. [#41432](https://github.com/el
+  ectron/electron/pull/41432)
+
+### Stack Changes
+
+- Chromium`126.0.6478.36`
+  - [New in 126](https://developer.chrome.com/blog/new-in-chrome-126/)
+  - [New in 125](https://developer.chrome.com/blog/new-in-chrome-125/)
+- Node `20.14.0`
+  - [Node 20.14.0 blog post](https://nodejs.org/en/blog/release/v20.14.0/)
+- V8 `12.6`
+
+Electron 31 upgrades Chromium from `124.0.6367.49` to `126.0.6478.36`, Node from `20.11.1` to `20.14.0`, and V8 from `12.4` to `12.6`.
+
+### New Features
+
+- Added `clearData` method to `Session`. [#40983](https://github.com/electron/electron/pull/40983)
+  - Added options parameter to `Session.clearData` API. [#41355](https://github.com/electron/electron/pull/41355)
+- Added support for Bluetooth ports being requested by service class ID in `navigator.serial`. [#41638](https://github.com/electron/electron/pull/41638)
+- Added support for Node's [`NODE_EXTRA_CA_CERTS`](https://nodejs.org/api/cli.html#node_extra_ca_certsfile) environment variable. [#41689](https://github.com/electron/electron/pull/41689)
+- Extended `webContents.setWindowOpenHandler` to support manual creation of BrowserWindow. [#41432](https://github.com/electron/electron/pull/41432)
+- Implemented support for the web standard [File System API](https://developer.mozilla.org/en-US/docs/Web/API/File_System_API). [#41419](https://github.com/electron/electron/pull/41419)
+- Extended `WebContentsView` to accept pre-existing `WebContents` instances. [#42319](https://github.com/electron/electron/pull/42319)
+- Added a new instance property `navigationHistory` on webContents API with `navigationHistory.getEntryAtIndex` method, enabling applications to retrieve the URL and title of any navigation entry within the browsing history. [#41577](https://github.com/electron/electron/pull/41577) <span style="font-size:small;">(Also in [29](https://github.com/electron/electron/pull/41661), [30](https://github.com/electron/electron/pull/41662))</span>
```

Comment:
issue(blocking): the website deploy is failing because it expects JSX-style properties, I think.

```suggestion
- Added a new instance property `navigationHistory` on webContents API with `navigationHistory.getEntryAtIndex` method, enabling applications to retrieve the URL and title of any navigation entry within the browsing history. [#41577](https://github.com/electron/electron/pull/41577) <span style={{fontSize: "small"}}>(Also in [29](https://github.com/electron/electron/pull/41661), [30](https://github.com/electron/electron/pull/41662))</span>
```

Reply:
Confirmed locally that this renders properly, will commit the suggestion.